web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -291,6 +292,12 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo }: TabbedDataGridProp
         setSnackbarSeverity('error')
         setSnackbarOpen(true)
       }
+    } else if (isUndefined(wf1Token)) {
```

Comment:
This will still be defined it will be expired, so the only way we'll know if it's expired is if we get a error response from the save forecast endpoint

Reply:
Ok thanks, that's what I figured